src/amberscript/parser.cc

```diff
@@ -22,6 +22,7 @@
 #include <utility>
 #include <vector>
 
+#include "amber/vulkan_header.h"
```

Comment:
I don't think you are supposed to include the Vulkan header in the parser. We cannot use Vulkan types here.

Reply:
Oh, I see. Makes sense.